content/developer/Further Resources.adoc

```diff
@@ -32,23 +32,23 @@ edition and much of the documentation is still valid. The appropriate
 version for SuiteCRM information is 6.5. Versions of documentation
 higher than this (i.e. 7) will probably not be relevant.
 
-* http://support.sugarcrm.com/02_Documentation/04_Sugar_Developer/[SugarCRM Developer docs]
-* http://developer.sugarcrm.com/[SugarCRM Developer Blog]
+* https://support.sugarcrm.com/Documentation/Sugar_Developer/[SugarCRM Developer docs]
```

Comment:
Maybe it's better to link to this one instead, since the current version has diverged too much:
https://support.sugarcrm.com/Documentation/Unsupported_Versions/Sugar_6.5/

Reply:
I updated the link and specified that it leads to version 6.5